requirements.txt

```diff
@@ -6,3 +6,4 @@ pyyaml==6.0.1
 pydantic==2.10.6
 numpy==2.2.3
 requests==2.32.3
+pip==24.3.1
```

Comment:
Should we omit this here? Seems like we shouldn't upgrade / downgrade pip itself when people do `pip install -r requirements.txt`?

Reply:
So just "pip" then, no version?

Reply:
yea let's go for that

Reply:
done!